models/discordactions.js

```diff
@@ -87,9 +90,37 @@ const addGroupRoleToMember = async (roleData) => {
   }
 };
 
+/**
+ *
+ * @param userDiscordId { String }: DiscordId of the user
+ * @returns {Promise<String>}
+ */
+const updateDiscordImageForVerification = async (userDiscordId) => {
+  try {
+    const discordAvatarUrl = await generateDiscordProfileImageUrl(userDiscordId);
+    const verificationDataSnapshot = await photoVerificationModel
+      .where("discordId", "==", userDiscordId)
+      .limit(1)
```

Comment:
if we are sure of having just 1 data as discussed in Voice call, then do we need this limit 1?

Reply:
done